[2024-10-01T17:07:09.732+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: third_task_test.compare_and_trigger manual__2024-10-01T17:07:06.588369+00:00 [queued]>
[2024-10-01T17:07:09.741+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: third_task_test.compare_and_trigger manual__2024-10-01T17:07:06.588369+00:00 [queued]>
[2024-10-01T17:07:09.742+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-10-01T17:07:09.756+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): compare_and_trigger> on 2024-10-01 17:07:06.588369+00:00
[2024-10-01T17:07:09.763+0000] {standard_task_runner.py:57} INFO - Started process 82 to run task
[2024-10-01T17:07:09.767+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'third_task_test', 'compare_and_trigger', 'manual__2024-10-01T17:07:06.588369+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/third_task.py', '--cfg-path', '/tmp/tmpwmh36nk0']
[2024-10-01T17:07:09.772+0000] {standard_task_runner.py:85} INFO - Job 100: Subtask compare_and_trigger
[2024-10-01T17:07:09.830+0000] {task_command.py:415} INFO - Running <TaskInstance: third_task_test.compare_and_trigger manual__2024-10-01T17:07:06.588369+00:00 [running]> on host f6937622df26
[2024-10-01T17:07:09.921+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='third_task_test' AIRFLOW_CTX_TASK_ID='compare_and_trigger' AIRFLOW_CTX_EXECUTION_DATE='2024-10-01T17:07:06.588369+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-01T17:07:06.588369+00:00'
[2024-10-01T17:07:09.924+0000] {logging_mixin.py:151} INFO - Attempting to read from bucket: huber-chatbot-project, key: sitemap_data/sitemap_data_2023.json
[2024-10-01T17:07:09.935+0000] {base.py:73} INFO - Using connection ID 'aws_default' for task execution.
[2024-10-01T17:07:09.936+0000] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-10-01T17:07:10.999+0000] {logging_mixin.py:151} INFO - All versions:
[2024-10-01T17:07:11.000+0000] {logging_mixin.py:151} INFO - There are not enough versions available.
[2024-10-01T17:07:11.002+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-10-01T17:07:11.016+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=third_task_test, task_id=compare_and_trigger, execution_date=20241001T170706, start_date=20241001T170709, end_date=20241001T170711
[2024-10-01T17:07:11.103+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-10-01T17:07:11.138+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
